packages/circularprogress/src/index.d.ts

```diff
@@ -0,0 +1,24 @@
+import { ForwardRefExoticComponent, RefAttributes } from 'react'
+
+interface Styling {
+  css?: Record<string, string | number>
+  style?: Record<string, string | number>
+  className?: string
+}
+
+export interface CircularProgressProps {
+  size?: 'small' | 'medium'
+  value?: number
+}
+
+type Sizes = {
+  small: 'small',
+  medium: 'medium'
+}
+
+export const sizes: Sizes
+
+declare const CircularProgress: ForwardRefExoticComponent<CircularProgressProps & Styling & RefAttributes<HTMLDivElement>>
```

Comment:
This took a bit of time to figure out the correct type. The biggest help was just seeing what `react` says is the return type of `React.forwardRef`